### DISCUSSION TITLE
Choice between Random or Rotate function

### DISCUSSION THREAD

**Discussion by DJMic** [AzuraCast/AzuraCast#7123] [Ideas]
It should be usefull to can choose between random or rotate function in the playlist playback mechanism. I know it's possible when using advanced playlists, butI would like to be able to use Azuracast's AutoDJ to take advantage of his features.

**Comment by LordHelmchen666** [comment]
What do you mean by random or rotate? There are already choosable options in playlists settings

**Song Playback Order**
- Shuffled
The full playlist is shuffled and then played through in the shuffled order.
- Random
A completely random track is picked for playback every time the queue is populated.
- Sequential
The order of the playlist is manually specified and followed by the AutoDJ.

**Reply by DJMic** [reply]
Rotations between differents playlists. Azuracast uses the Random function of liquidsoap. But it's not precise and I don't want use Advanced Playlist to use the Rotate function of liquidsoap. It's more user-friendly and easier to manage programming with an interface rather than having to write code and avoid restarting liquidsoap for the configuration to take effect. In addition, by using advanced playlists, you lose the functionalities of AutoDJ.
